### ISSUE TITLE
Rassembler les opérations (aggregates) de mongodb dans une classe

### ISSUE THREAD

**Issue by evaletolab** [karibou-ch/karibou-quad#13]
Le découplage avec mongodb est important ( karibou.ch utilise mongodb 3.0) Pour la suite, je veux pouvoir faire les transformations que tu utilises sans passer par mongo.  Il convient alors de créer une classe qui contient tous les `aggrégates` nécessaires (comme provider de données)

**Comment by jcavat** [comment]
Si tu changes de fonction de mapping/db il suffit de créer un autre serveur REST avec les mêmes routes.

**Comment by evaletolab** [comment]
D'expérience, je tiens au MVC (like), je souhaite une classe qui expose les données
voir discussion  #12

**Comment by jcavat** [comment]
Oui, c'est ce que j'ai dit dans #12. A terme, j'aimerais mieux représenter la structure d'une réponse.
